interface/forms_admin/forms_admin.php

```diff
@@ -32,16 +38,8 @@
 $bigdata = getRegistered("%") or $bigdata = false;
 
 //START OUT OUR PAGE....
-?>
-<html>
-<head>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-</head>
-<body class="body_top">
-<span class="title"><?php xl('Forms Administration','e');?></span>
-<br><br>
-<?php
+require_once "{$GLOBALS['srcdir']}/templates/standard_header_template.php";
```

Comment:
since there is no head html element in this script anymore, shouldn't this be included within the twig template itself?